hack/test-update-storage-objects.sh

```diff
@@ -112,10 +112,10 @@ tests=(
 examples/persistent-volume-provisioning/rbd/rbd-storage-class.yaml,storageclasses,,slow,v1beta1,v1
 )
 
-KUBE_OLD_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1beta1,extensions/v1beta1"
-KUBE_NEW_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1,extensions/v1beta1"
-KUBE_OLD_STORAGE_VERSIONS="storage.k8s.io/v1beta1"
-KUBE_NEW_STORAGE_VERSIONS="storage.k8s.io/v1"
+KUBE_OLD_API_VERSION="storage.k8s.io/v1beta1,extensions/v1beta1"
+KUBE_NEW_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1"
+KUBE_OLD_STORAGE_VERSIONS="extensions/v1beta1,storage.k8s.io/v1beta1"
```

Comment:
I am not familiar enough with this file to comment on if this makes sense.

Reply:
I also have no idea about this file, though I did expect networkpolicies to be added to the list to migrate in update-storage-objects.sh

Reply:
You also need to add an example networkingpolicy object at https://github.com/danwinship/kubernetes/blob/9468b49ad6e010aebaaeccaf412ba39f17726ff0/hack/test-update-storage-objects.sh#L112. Otherwise it's not tested.

Reply:
Yup - we shouldn't remove it and we should add an object of that type to actually test it.

Reply:
@wojtek-t we shouldn't remove *what*?

Reply:
@danwinship, could you confirm if this test catches the bug *after* you add a test type, but *without* fixing hack/upgrade-storage-objects.sh?

[edit] I see your latest comments confirmed the failure. That's good.